### PR TITLE
Feacture/video

### PR DESCRIPTION
**Pull request by joanafbrito**
# Description

We Added a Video trailer from the api , to the Chosen One. Made sure the video was working for different movies.

next steps :
will will need to figure it out how the Vimeo URL should be for the videos to show in our page, or add as a second condition on the filter helper Function to show only Youtube videos.
For now the videos from Youtube are perfect!

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [x]Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [x] dev tools
- [ ] npm test

# Checklist:

- [ ] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,11 @@
 
 ## A Little bit about the Developer
 
-*Hi! My name is Joana and I am a current Front End Engineering student at Turing School of Software Design. I am a career changer! I've  spent the last decade working to manage two small business, and grabbing the opportunity to move to another country. 
-Now, I'm spending my days learning code. As a woman I can contribuit to build a more balanced society, where people can be seen as equals in the software marketing and where I can build apps more inclusive and make a difference using this huge platform of the internet!*
+*Hi! My name is Joana and I am a current Front End Engineering student at Turing School of Software Design. I am a career changer! I've  spent the last decade working to manage two small business, and grabbing the opportunity to move to another country.*
```
Comment (Meggs625) [inline README.md:14]: Thank you for adding this information :)

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "prop-types": "^15.7.2",
```
Comment (Meggs625) [inline package.json:9]: Looking forward to adding the prop-types

File: src/App.js [hunk 3]
```diff
@@ -13,49 +12,51 @@ class App extends Component {
...
+
     return (
       null
     );
-    // const movieById = this.state.movies.find(movie => movie.id === movieId)
-    // this.setState( {selectedMovie: movieById})
+
+  }
+
+  filterVideoByType = (dataVideos) => {
```
Comment (Meggs625) [inline src/App.js:44]: Good filter

File: src/ChosenOne.js [hunk 4]
```diff
@@ -1,19 +1,44 @@
...
+      />
+    </div>
     <h2>{props.details.overview}</h2>
     <img className='movie-pic'src={props.details.backdrop_path} alt='movie poster' />
   </div>
 )
 
 }
+
+// Pseudo-code for videos from vimeo - not sure if we will have time to do that
```
Comment (Meggs625) [inline src/ChosenOne.js:32]: This will definitely be helpful when we come back and wonder what we were thinking about this :)

**Review by Meggs625 — APPROVED** [review]
Great job! I'm so excited that we got the video to work!!

File: package.json (hunk 2 above)
Comment (joanafbrito) [reply package.json:9]: me too!

**Pull request merged**